mlflow/store/tracking/sqlalchemy_store.py

```diff
@@ -1185,7 +1175,7 @@ def _try_insert_tags(attempt_number, max_retries):
                             new_tag_dict[tag.key] = new_tag
 
                     # finally, save new entries to DB.
-                    self._save_to_db(session=session, objs=list(new_tag_dict.values()))
+                    session.add_all(new_tag_dict.values())
```

Comment:
Does this work? .values() is not a list right?

Reply:
This should work because `values` is an iterator.

Reply:
Ok!